package.json

```diff
@@ -49,14 +49,16 @@
     "jsdom": "^9.11.0",
     "lerna": "2.0.0-beta.38",
     "prettier": "^0.22.0",
+    "prop-types": "^15.5.6",
     "react": "^15.0.0-0",
     "react-addons-css-transition-group": "^15.0.0-0",
-    "react-addons-test-utils": "^15.4.2",
+    "react-addons-test-utils": "^15.5.1",
```

Comment:
If we only ever test with React 15.5, we should remove this because https://facebook.github.io/react/blog/#react-test-utils